backend/config/__init__.py

```diff
@@ -1,4 +1,7 @@
 import os
+from dotenv import find_dotenv, load_dotenv
+
+load_dotenv(find_dotenv())
```

Comment:
How does this works? Even having the ``.env`` file in the root of the project this library is going to find it?

Reply:
Exactly! I can actually simplify this a bit by removing `find_dotenv`, as `load_dotenv` calls it internally (if no path is given). But essentially it looks for a file called `.env` in the root directly, then starts working it's way down.

Reply:
Ah I see. But how it finds the .env file if it is placed in the parent directory of backend directory?

Reply:
✨ _Magic_ ✨ 

In all seriousness, seems I read the source wrong; it actually walks **up** to the project's root looking for `.env`. So you can call `python main.py` while in the `backend` directory and it'll still find it.

https://github.com/theskumar/python-dotenv/blob/137bc3dc0b8cf3d417a1e800c4065c526e3fb96a/src/dotenv/main.py#L252